detekt-rules-errorprone/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/bugs/CastNullableToNonNullableType.kt

```diff
@@ -48,6 +50,9 @@ class CastNullableToNonNullableType(config: Config = Config.empty) : Rule(config
         Debt.FIVE_MINS
     )
 
+    @Configuration("Whether platform types should be considered as non-nullable and ignored by this rule")
+    private val ignorePlatformType: Boolean by config(true)
```

Comment:
Should this be plural?
```suggestion
    private val ignorePlatformTypes: Boolean by config(true)
```

Reply:
Updated